src/Api/Endpoints/Apps.cs

```diff
@@ -136,6 +137,64 @@ public static void MapAccountEndpoints(this WebApplication app)
             .WithParameterValidation()
             .RequireManagementKey()
             .RequireCors("default");
+
+        app.MapGet("/apps/list-authenticators", ListConfiguredAuthenticatorsAsync)
+            .RequireSecretKey()
+            .RequireCors();
```

Comment:
issue: This breaks the existing pattern of using `/list`. Is there an argument to be made to stray away from the convention?

suggestion: change this api pattern to follow convention`/apps/authenticators/list`

thought: I think we should do a team call to discuss API design for the `authenticators`-API. Can you schedule it?